File: src/BFF/BFF/Models/Oefening.cs
using System;
using System.Collections.Generic;
using BFF.Enums;

namespace BFF.Models
{
    public class Oefening
    {
        public Guid Id { get; set; }
        public string Naam { get; set; }
        public string Omschrijving { get; set; }
        public ICollection<Prestatie> Prestaties { get; set; }
        public Spiergroep? Spiergroep { get; set; }
    }
}

File: src/FrontendService/src/app/models/oefening.ts
import { Spiergroep } from '../enums/spiergroep';
import { PrestatieDag } from './prestatie-dag';

export class Oefening {
    id: string;
    naam: string;
    omschrijving: string;
    spiergroep: Spiergroep;
    prestatieDagen: PrestatieDag[];
}